tools/depends/target/python3/darwin_embedded.patch

```diff
@@ -231,11 +232,11 @@
 +#  undef HAVE_WAIT
 +#  undef HAVE_WAIT3
 +#  undef HAVE_WAIT4
-+#  undef HAVE_WAITPID
++//#  undef HAVE_WAITPID
```

Comment:
Why patch commented `#  undef HAVE_WAITPID` in, instead of not patching this line in?

Reply:
at the time was just easier to update the patch by modifying the comment rather than rediff it all. I'll get it removed proper now